### PR TITLE
updated styling

### PR DESCRIPTION
**Pull request by tusharkandpal**
- proper import of styles
- created component.css
- organizing for better use case

### CODE REVIEW

**Comment by ozairkhan2810** [comment]
Code looks clean. One suggestion, you could place all the root properties in one file.

File: components/alert/alert.css [hunk 1]
```diff
@@ -1,82 +1,68 @@
...
+  --success-alert-light: #d1e7dd;
+  --danger-alert-light: #f8d7da;
+  --warning-alert-light: #fff3cd;
+  --info-alert-light: #cff4fc;
+  --primary-alert-dark: #2864be;
+  --secondary-alert-dark: #808080;
+  --success-alert-dark: #17925a;
+  --danger-alert-dark: #ca2b39;
+  --warning-alert-dark: #b89628;
+  --info-alert-dark: #2d96ad;
```
Comment (JantuDeb) [inline components/alert/alert.css:14]: Avoid making super-specific CSS variables. You may reuse primary, success, and warning colors in other places.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: components/alert/alert.css (hunk 1 above)
Comment (tusharkandpal) [reply components/alert/alert.css:14]: But i wanted a different shade of it. That is why.

**Review by ozairkhan2810 — APPROVED** [review]

**Comment by tusharkandpal** [comment]
> Code looks clean. One suggestion, you could place all the root properties in one file.

Im keeping colors based on the component to easily identify them

**Pull request merged**